src/code/audio/code_8019AF00.c

```diff
@@ -154,7 +154,7 @@ OcarinaNote sScarecrowsLongSongSecondNote;
 u16 sCustomSequencePc;
 
 // Sfx Data
-u8 D_801D6600[] = {
+u8 gSfxBankHasMoreThan255Entries[] = {
```

Comment:
Hmm, I'm not really a big fan of the name, though don't really know enough to suggest a better one. I see one usage down below. Is that the only one or are there others?

Reply:
Oh right. OoT has this variable called `gIsLargeSfxBank` which is a much cleaner name, but it took me forever to figure out what it means. It's specifically because the sequence needs to store the ID in a `u8` value, but can only hold `0xFF` values. So if the number of sfx is greater than 255, then the sequence needs two `u8` to store the lower and upper bits. Perhaps this is better explained in comments instead of a verbose temp name, and I can just stick with `gIsLargeSfxBank`

Reply:
Personally I like `gIsLargeSfxBank` with the comment. What you have just seems a bit too on the nose for me.